README.md

```diff
@@ -8,8 +8,11 @@ You can open [issues](https://github.com/Azure/AppConfiguration/issues?utf8=%E2%
 Subscribe to the following repo to be notified of announcements and updates about Azure App Configuration.
   * [Azure/AppConfiguration-Announcements](https://github.com/Azure/AppConfiguration-Announcements)
 
+## Roadmap
+Check out what is on the [roadmap](https://github.com/Azure/AppConfiguration/projects/1) of Azure App Configuration and what the team is working on.
```

Comment:
This is a nitpick: "what the team is currently working on"

Reply:
Updated. Thanks!